PermissionsUtilsUI/src/main/res/layout/activity_app_diagnosis.xml

```diff
@@ -0,0 +1,242 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".diagnosis.activity.AppDiagnosisActivity">
+
+    <android.support.constraint.ConstraintLayout
+        android:id="@+id/Constraint"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <ImageView
+            android:id="@+id/image_view_summary_icon"
+            android:layout_width="@dimen/dk_ic_xxbig"
+            android:layout_height="@dimen/dk_ic_xxbig"
+            android:layout_margin="@dimen/dk_margin_half"
+            android:layout_marginBottom="@dimen/dk_margin"
+            android:padding="@dimen/dk_margin_quarter"
+            android:src="@drawable/ic_checked"
```

Comment:
@mohamed-garci All drawables must follow name guideline like : "dk_vehicle_xxxxx"
Also, drawables have to be set dynamically with `DKResources`

Reply:
Done